code/ship/ship.cpp

```diff
@@ -6789,27 +6789,10 @@ void ship_render_cockpit(object *objp)
 
 void ship_render_show_ship_cockpit(object *objp)
 {
-	if (objp->type != OBJ_SHIP || objp->instance < 0)
```

Comment:
Since this function must operate on a ship, you should keep this if() condition.

Reply:
Good point. I added it in with the previous PR so I will add it back in this one, too.